src/Bicep.Core/TypeSystem/DeployTimeConstantVariableVisitor.cs

```diff
@@ -13,25 +13,27 @@ namespace Bicep.Core.TypeSystem
     public sealed class DeployTimeConstantVariableVisitor : SyntaxVisitor
     {
         private readonly SemanticModel model;
-        public Stack<string> visitedStack;
-        public ObjectType? invalidReferencedBodyObj;
-
+        
         public DeployTimeConstantVariableVisitor(SemanticModel model)
         {
             this.model = model;
-            this.visitedStack = new Stack<string>();
+            this.VisitedStack = new Stack<string>();
         }
 
+        public Stack<string> VisitedStack { get; private set; }
```

Comment:
Should we remove `private set;` since it's only initialized in the constructor?

Reply:
Sure - removed it.